sql/postgresql/PostgreSQLParser.g4

```diff
@@ -325,7 +325,7 @@ iso_level
     | SERIALIZABLE
     ;
 
-opt_boolean_or_string
+boolean_or_string_
```

Comment:
`opt_boolean_or_string` was renamed by "rename.sh" because the symbol is *not* nullable. It's completely unclear why it was named with a leading "opt_"!! The Bison grammar has problems.